src/app/api/utils/featureFlags.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import appConstants from "../../../appConstants"
+
+type FeatureFlags = {
+  [key: string]: FeatureFlag | boolean
+}
+
+type FeatureFlag = {
+  name?: string
+  description?: string
+  dependencies?: [string]
+}
+
+const featureFlags = JSON.parse(appConstants.FEATURE_FLAGS) as FeatureFlags
+
+export function checkFeatureFlag(flag: string) : (FeatureFlag | boolean) {
```

Comment:
Hmm, could we rename this to `isFeatureFlagEnabled` and make it return a boolean? We could always add a `getFeatureFlagDetails` if we need all that.

(Also, maybe the `FeatureFlag` type could also add an `enabled` boolean property? So that we can (temporarily) toggle it off without removing the reference to it.)